app/components/appMainModule/committeePageModule/committeePage.html

```diff
@@ -1,5 +1,5 @@
 
-<div class="container">
+<div class="fluid-container">
```

Comment:
This isn't a real class, you're looking for `container-fluid`, but there's a few more changes to make the background bleed to the edge which I addressed in https://github.com/caciviclab/disclosure-frontend/pull/221 so don't worry about it.